Sources/SchafKit/Extensions/Collections/Array.swift

```diff
@@ -108,8 +108,8 @@ public extension Array {
     /// ```
     ///
     /// - Parameter f: A function/keypath returning the value to be compared.
```

Comment:
Please also change the parameter name here.

Reply:
Done

Reply:
No